pegasus/sites.v3/code.org/public/js/about_headshots.js

```diff
@@ -0,0 +1,32 @@
+window.addEventListener('resize', function () {
+    window.location.reload();
```

Comment:
This seems heavy-handed.  Normally we would throttle responses to resize (in case a user is rapidly dragging the window size) but then just reapply the layout rather than reload the whole window.